modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/model/artifact/facts/ArtifactCPE.java

```diff
@@ -9,7 +9,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  */
-package org.eclipse.sw360.antenna.model.artifact.facts;
+package org.eclipse.sw360.antenna.sw360.model.artifact.facts;
```

Comment:
This is not SW360 specific information, although it currently is only used by SW360. Maybe that could be merged with `Issues`/`SecurityIssues`.